src/App.js

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
+import Sum from './components/Sum';
+
 import './App.scss';
 
 const App = () => (
-  <h1>React sum</h1>
+  <>
+    <h1>React sum</h1>
+    <Sum
+      a={2}
+      b={2}
+    />
```

Comment:
No need to add extra lines here, code is short enough.
```suggestion
    <Sum a={2} b={2} />
```